glide.yaml

```diff
@@ -12,3 +12,5 @@ import:
   - go/ast/astutil
 - package: github.com/jessevdk/go-flags
 - package: github.com/anmitsu/go-shlex
+- package: go.uber.org/multierr
+  version: ~0.2.0
```

Comment:
Yikes - taking a hard dep on a pre 1.0 lib, in a 1.0 lib...

Doesn't this mean we need to 1.0 multierr now?

@abhinav @prashantv lets jam in person - I don't think I'm mistaken.